jormungandr/src/secure/mod.rs

```diff
@@ -29,7 +29,9 @@ pub struct GenesisPraos {
 ///
 #[derive(Clone, Deserialize)]
 pub struct GenesisPraosPublic {
+    #[allow(dead_code)]
```

Comment:
Why is this dead code now?

Reply:
I was too afraid to remove it :) But then noticed that there is a PR on master which is removing them. so I'll do the same